src/Panel/EnvironmentPanel.php

```diff
@@ -48,6 +48,7 @@ protected function _prepare()
             'CACHE' => CACHE,
             'CAKE' => CAKE,
             'CAKE_CORE_INCLUDE_PATH' => CAKE_CORE_INCLUDE_PATH,
+            'CONFIG' => CONFIG,
```

Comment:
But then only DS, TIME_START and time constants are _define()_'ed withing cakephp/cakephp (except `tests/bootstrap.php`).

Every other constant is defined in `cakephp/app` bootstrap.php.

CakePHP depends on CONFIG in multiple places, so it should be core constant.

It listed as core constant in [docs](https://book.cakephp.org/3.0/en/core-libraries/global-constants-and-functions.html#core-definition-constants)